package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oat-sa/tao-core-sdk",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Could you please add `src` into the `files` array. I know it was my fault to forget to add, but without that we won't be able to use this package in ES6 environment.